### PR TITLE
fix(cb2-6727): fix permission errors with config file

### PR DESCRIPTION
**Pull request by naathanbrown**
## Ticket title

The config file had errors inside, this repairs them.
[https://dvsa.atlassian.net/browse/CB2-6727](6727)

## Checklist

- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by aaron-downing — APPROVED** [review]

File: src/functions/functionalConfig.ts [hunk 1]
```diff
@@ -43,17 +43,17 @@ export const functionConfig: { [key: string]: NonEmptyArray<IApiAccess> } = {
...
   "TestResult.CreateContingency": [
     {
       verbs: ["POST", "OPTIONS"],
-      path: "test-result/*",
+      path: "test-results/*",
     },
     {
       verbs: ["GET", "OPTIONS"],
-      path: "test-types/*",
+      path: "test-types",
```
Comment (Twebb97) [inline src/functions/functionalConfig.ts:56]: you need both test-types and test-types/*

**Review by Twebb97 — CHANGES_REQUESTED** [review]
nice naming refactor

File: src/functions/functionalConfig.ts (hunk 1 above)
Comment (aaron-downing) [reply src/functions/functionalConfig.ts:56]: Think Nathan tested it and it worked fine

Comment (aaron-downing) [reply src/functions/functionalConfig.ts:56]: test-types doesn't have any endpoints under test-types/

Comment (Twebb97) [reply src/functions/functionalConfig.ts:60]: it does

Comment (Twebb97) [reply src/functions/functionalConfig.ts:60]: https://api.develop.cvs.dvsacloud.uk/develop/test-types/:id

**Review by aaron-downing — APPROVED** [review]

**Pull request merged**